src/main/java/morgana/commands/FindCommand.java

```diff
@@ -0,0 +1,38 @@
+package morgana.commands;
+
+import morgana.exceptions.MorganaException;
+import morgana.storage.Storage;
+import morgana.task.Task;
+import morgana.task.TaskList;
+import morgana.ui.Ui;
+
+/**
+ * Represents a command to find tasks that match a given keyword in their description.
+ */
+public class FindCommand extends Command {
+    private final String keyword;
```

Comment:
Consider changing the keyword to CAPS, I have also seen the issue elsewhere in the code